languages/ruby/reference/exercise-concepts/resistor-color-duo.md

```diff
@@ -0,0 +1,35 @@
+# Concepts of resistor-color-duo
+
+## General
+
+ [Example solution](https://github.com/exercism/ruby/blob/master/exercises/resistor-color-duo/.meta/solutions/resistor_color_duo.rb)
+
+- Classes: defining classes
+- Methods: arguments/parameters, defining methods and return values
+- Strings: Basics like difference between `1` and `"1"`
+- Class methods: How to define a method so it can be called directly on a class
+- Constants: How to create, name and use constants
+
+Depending on the approach either:
+- Hashes: How to initialize/create a hash, how to access it's values
+
+or
+- Arrays: How to initialize/create an array, what are indexes and how can they be useful, how to get the index of a value that's stored in the array
```

Comment:
These are bullets, and can have sub-bullets.

- Arrays
  - How to initialize/create an array
  - What is an index
    - Why are they useful?
  - How to get the index of a value in an array

Reply:
This feedback applies to other areas of this document.

Reply:
Thanks for the review! Fixed the formatting now :)

Github was down for some time and changes to my branch weren't updated in the PR, not sure why. Had to do another push to make it work.